nifi-nar-bundles/nifi-standard-services/nifi-lookup-services-bundle/nifi-lookup-services/src/main/java/org/apache/nifi/lookup/SimpleKeyValueLookupService.java

```diff
@@ -47,6 +47,7 @@ protected PropertyDescriptor getSupportedDynamicPropertyDescriptor(final String
             .required(false)
             .dynamic(true)
             .addValidator(Validator.VALID)
+            .expressionLanguageSupported(true)
```

Comment:
I think supporting EL is useful, because user can refer variable registries or do some text processing with EL.